JPS_VIRTUALSENSOR/ShipInputAgent/src/main/java/com/cmclinnovations/ship/QueryClient.java

```diff
@@ -416,31 +417,22 @@ void setMeasureIri(List<Ship> ships) {
 
     void updateTimeSeriesData(List<Ship> ships) {
         try (Connection conn = remoteRDBStoreClient.getConnection()) {
-            ships.stream().forEach(ship -> {
+            ships.stream().filter(ship -> ship.hasTimeSeries() && !ship.getTimestampList().isEmpty()).forEach(ship -> {
```

Comment:
This change breaks the case where ship data is provided by static JSON file. Please change the logic to accommodate both static JSON and live data stream cases.

Reply:
the relevant commit is reverted, should be fixed